cmd/bnsd/client/testdata/genesis.json

```diff
@@ -33,12 +33,15 @@
   },
   "chain_id": "test-chain-KqKNbF",
   "consensus_params": {
-    "block_size_params": {
+    "block_size": {
       "max_bytes": "22020096",
       "max_gas": "-1"
     },
-    "evidence_params": {
+    "evidence": {
       "max_age": "100000"
+    },
+    "validator": {
```

Comment:
nice one.
is there docs on all this?

Reply:
There are some mentions in the [release changelog](https://github.com/tendermint/tendermint/blob/master/CHANGELOG.md#0274), but I was just looking at the code to figure this out.
Documentation is mentioning some of those fields, but it does not explain where they are in the genesis file, which makes the whole documentation useless.